### ISSUE TITLE
client: `eth_sendRawTransaction` response misleading when client isn't connected to other peers

### ISSUE THREAD

**Issue by acolytec3** [ethereumjs/ethereumjs-monorepo#1435]
The RPC response for `eth_sendRawTransaction` currently responds like below when the client is connected to peers and accepts a transaction submitted via RPC.
```
{"jsonrpc":"2.0","id":1,"result":"0xae9707c0a61a21816f46d5b3c40747946a5f7af1ecf5e8fcb34d1c082145b305"}
```

It generates the below response when not connected to other peers. 

```
{"jsonrpc":"2.0","id":1,"result":{"code":-32603,"message":"no peer connection available"}}
```

 This behavior made sense when our client didn't have a transaction pool but should be reworked now that we are inserting new transactions submitted via RPC into our own transaction pool and then broadcasting when connected to other peers.

**Comment by holgerd77** [comment]
Has this been addressed at some point respectively is this still an issue?

**Comment by ryanio** [comment]
Looks like I solved this as part of #1519, in sendRawTransaction there is now a check that `!this.client.config.mine` : 
https://github.com/ethereumjs/ethereumjs-monorepo/blob/81c2bf08d4946a3a24c8d5c0ea505c8c74ae31ed/packages/client/lib/rpc/modules/eth.ts#L899-L908
